### PR TITLE
CI dont fail import scipy.stats

### PR DESCRIPTION
**Pull request by MarcoGorelli**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/util/_test_decorators.py [hunk 1]
```diff
@@ -77,6 +77,13 @@ def safe_import(mod_name: str, min_version: str | None = None):
             return False
         else:
             raise
+    except AttributeError:
+        # TODO remove once https://github.com/scipy/scipy/issues/17827
+        # is addressed
```
Comment (simonjayhawkins) [inline pandas/util/_test_decorators.py:82]: do we need a pandas issue to track this? (that scipy issue is closed as duplicate)

**Review by phofl — APPROVED** [review]
lets open an issue on our side as well

**Comment by jbrockmendel** [comment]
scipy#17827 is closed.  does that solve it on our end?

**Comment by mroeschke** [comment]
If I am reading https://github.com/numpy/numpy/pull/23034 correctly, this may be fixed on the next numpy nightly release and we won't have to do anything on our end.

**Comment by MarcoGorelli** [comment]
> If I am reading [numpy/numpy#23034](https://github.com/numpy/numpy/pull/23034) correctly, this may be fixed on the next numpy nightly release and we won't have to do anything on our end.

the best kind of fix 🎉 ok cool, closing then, hopefully this'll have sorted itself out tomorrow

**Pull request closed without merging**